static/app/views/performance/transactionSummary/transactionEvents/operationSort.tsx

```diff
@@ -0,0 +1,289 @@
+import {Manager, Popper, Reference} from 'react-popper';
+import styled from '@emotion/styled';
+import {Location, LocationDescriptorObject} from 'history';
+import * as PopperJS from 'popper.js';
+
+import {GetActorPropsFn} from 'app/components/dropdownMenu';
+import MenuItem from 'app/components/menuItem';
+import Radio from 'app/components/radio';
+import {TableData} from 'app/utils/discover/discoverQuery';
+import EventView from 'app/utils/discover/eventView';
+
+export type TitleProps = Partial<ReturnType<GetActorPropsFn>>;
+
+type Props = {
+  title: ComponentClass<TitleProps>;
+  eventView: EventView;
+  tableMeta: TableData['meta'];
+  location: Location;
+};
+
+type State = {
+  isOpen: boolean;
+};
+
+class OperationSort extends Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+
+    let portal = document.getElementById('team-key-transaction-portal');
+    if (!portal) {
+      portal = document.createElement('div');
+      portal.setAttribute('id', 'team-key-transaction-portal');
+      document.body.appendChild(portal);
+    }
+    this.portalEl = portal;
+    this.menuEl = null;
+  }
+
+  state: State = {
+    isOpen: false,
+  };
+
+  componentDidUpdate(_props: Props, prevState: State) {
+    if (this.state.isOpen && prevState.isOpen === false) {
+      document.addEventListener('click', this.handleClickOutside, true);
+    }
+    if (this.state.isOpen === false && prevState.isOpen) {
+      document.removeEventListener('click', this.handleClickOutside, true);
+    }
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('click', this.handleClickOutside, true);
+    this.portalEl.remove();
+  }
+
+  private portalEl: Element;
+  private menuEl: Element | null;
+
+  handleClickOutside = (event: MouseEvent) => {
+    if (!this.menuEl) {
+      return;
+    }
+    if (!(event.target instanceof Element)) {
+      return;
+    }
+    if (this.menuEl.contains(event.target)) {
+      return;
+    }
+    this.setState({isOpen: false});
+  };
+
+  toggleOpen = () => {
+    this.setState(({isOpen}) => ({isOpen: !isOpen}));
+  };
+
+  renderMenuContent() {
+    const {eventView, tableMeta, location} = this.props;
+
+    function generateSortLink(field): LocationDescriptorObject | undefined {
+      if (!tableMeta) {
+        return undefined;
+      }
+
+      const nextEventView = eventView.sortOnField(field, tableMeta, 'desc');
+      const queryStringObject = nextEventView.generateQueryStringObject();
+
+      return {
+        ...location,
+        query: {...location.query, sort: queryStringObject.sort},
+      };
+    }
+
+    return (
+      <DropdownContent>
+        <DropdownMenuItem to={generateSortLink({field: 'spans.http'})} stopPropagation>
+          <MenuItemContent>
+            <Radio
+              radioSize="small"
+              onChange={() => {}}
+              checked={location.query.sort === '-spans.http'}
+            />
+            <MenuItemText>Sort By HTTP</MenuItemText>
```

Comment:
You'll want to wrap all the other ones as well

```suggestion
            <MenuItemText>{t('Sort By HTTP')}</MenuItemText>
```

Reply:
Wrapped!